space/space.go

```diff
@@ -312,6 +313,29 @@ func (r *GormRepository) Create(ctx context.Context, space *Space) (*Space, erro
 	return space, nil
 }
 
+// Delete removes a single record. This method use custom SQL to allow soft delete with GORM
+// to coexist with permanent delete.
+func (m *GormRepository) PermanentDelete(ctx context.Context, ID uuid.UUID) error {
+	defer goa.MeasureSince([]string{"goa", "db", "space", "permanent_delete"}, time.Now())
+	tx := m.db.Unscoped().Delete(&Space{ID: ID})
+	if err := tx.Error; err != nil {
+		log.Error(ctx, map[string]interface{}{
+			"space_id": ID.String(),
+		}, "unable to permanently delete the space")
+		return errors.NewInternalError(ctx, err)
+	}
+	if tx.RowsAffected == 0 {
+		log.Error(ctx, map[string]interface{}{
+			"space_id": ID.String(),
+		}, "none row was affected by the permanently deletion operation")
+		return errors.NewNotFoundError("space", ID.String())
+	}
+	log.Debug(ctx, map[string]interface{}{
```

Comment:
I wonder if this log message is logged in production with this level or if it is helping at all.